src/codeu/chat/client/simplegui/ChatGuiFX.java

```diff
@@ -32,7 +33,7 @@
     private Scene signInScene, mainScene;           // Scenes to hold all the elements for each page
     private TextField userInput;
     private PasswordField passInput;                // Takes input for username and password
-    private ClientContext clientContext;            //
+    private static ClientContext clientContext;            //
```

Comment:
This is a result of the problem with two instances I was talking about in the chat. Note that using the `static` keyword is a bit of a hack. Malik and I talked about this already, so hopefully he's not too offended by that! But if at all possible you're going to want to get rid of this static and fix the underlying problem of creating two instances of the GUI.